source/includes/_eos.md

```diff
@@ -33,35 +33,35 @@ Before starting, it is necessary to include the `eos_api.hpp` header file. This
 It is highly recommended to always use the latest version.
 
 ```c++
-#include <eosiolib/eosio.hpp>                       
-#include <eosiolib/print.hpp>                       
-
-#include "oraclize/eos_api.hpp"                  
+#include "oraclize/eos_api.hpp"
 
 using namespace eosio;
 
-
-class example1 : public eosio::contract {
+class eosusdprice : public eosio::contract 
+{
   public:
       using contract::contract;
 
-      /// @abi action
-      void getprice() {
-         oraclize_query("URL", "json(https://api.binance.com/api/v3/ticker/price?symbol=EOSUSDT).price");               
-         print("Oraclize query was sent, standing by for the answer..");                          
-      }
+      eosusdprice(name receiver, name code, datastream<const char*> ds) : contract(receiver, code, ds) {}
 
-      /// @abi action
-      void callback( checksum256 queryId, std::vector<unsigned char> result, std::vector<unsigned char> proof ) {
-         require_auth(oraclize_cbAddress());
-         
-         std::string result_str = vector_to_string(result);                                              
-         print("EOSUSD:", result_str);
+      [[eosio::action]]
+      void execquery() {
+          print("Sending query to Oraclize...");
+          oraclize_query("URL", "json(https://min-api.cryptocompare.com/data/price?fsym=EOS&tsyms=USD).USD",\
```

Comment:
I actually prefer to use half the indentation when the code is line-broken, but I am not sure if this is a stylistic best practice. Anyway I would leave this comment like this since highlights the proof declaration: 
![image](https://user-images.githubusercontent.com/10484858/52337096-9df4aa00-29fe-11e9-9c49-2e45dc64a296.png)